pkg/service/webhook/service.go

```diff
@@ -173,13 +175,14 @@ func (s Service) GetSupportedVerbs() GetSupportedVerbsResponse {
 	return GetSupportedVerbsResponse{Verbs: []Verb{Create, Delete}}
 }
 
-func (s Service) PublishWebhook(noun Noun, verb Verb, payloadReader io.Reader) {
-	ctx, cancel := context.WithTimeout(context.Background(), s.timeoutDuration)
+// TODO: consider returning an error to be handled by the gin middleware
```

Comment:
Be careful I tried thee same thing here, the c Context finishes during the middle of the PublishWebhook call and it never does a POST at the bottom, can do an integration test by creaing a nodejs server to recieve webhook, or can pull down this branch of playground to test webhook - https://github.com/TBD54566975/ssi-playground/tree/webhook-demo

Reply:
It doesn't cover it because this is a cancellation issue. You'd need to time things perfectly. 

The test could be modified to make it happen, but I don't this it's worth it.